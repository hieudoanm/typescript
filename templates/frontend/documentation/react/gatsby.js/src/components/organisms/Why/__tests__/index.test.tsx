import { render } from '@testing-library/react';
import { Why } from '../index';
import React from 'react';

describe('Why', () => {
  it('renders correctly', () => {
    const button = render(<Why reasons={[]} />);
    expect(button.container).toMatchSnapshot();
  });
});
