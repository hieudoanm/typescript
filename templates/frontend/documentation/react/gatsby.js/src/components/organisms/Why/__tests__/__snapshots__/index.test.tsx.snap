// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Why renders correctly 1`] = `
<div>
  <section
    class="why-container py-8 md:py-16"
    id="why"
  >
    <div
      class="container mx-auto"
    >
      <div
        class="tabs"
      />
    </div>
  </section>
</div>
`;
